src/libs/elektra/keyset.c

```diff
@@ -709,6 +709,38 @@ ssize_t ksSearchInternal (const KeySet * ks, const Key * toAppend)
 	}
 }
 
+/**
+ * Efficient search in a key set, either yielding the actual index
+ * of the key, if the ley has been found within the key set, or a
+ * negative value indicating the insertion index of the key, if the
+ * key would be inserted.
+ *
+ * @code
+
+ssize_t result = ksSearch(ks, key);
+
+if (result >= 0)
+{
+	ssize_t position = result;
+	// Seems like the key already exist.
```

Comment:
we assert that:

```suggestion
	// The key already exist in kst.
```

Reply:
This was inherited from `ksSearchInternal` and has been updated in both function docs.